### ISSUE TITLE
Global Voice Chat.

### ISSUE THREAD

**Issue by Xelor41023** [bitfighter/bitfighter#539]
Consider implementing a Global Voice Chat in addition to the team one.... Very little effort required and Would help make the game more enjoyable....

**Comment by NothingMuchHereToSay** [comment]
Yeah, I think the problem with that is there IS a global voice chat... for the same team.

For example: When you play Bitmatch, all ships are blue, and you can talk to each other. However, when playing anything team-based, you get an automatic team voice chat.

**Comment by fordcars** [comment]
I'd want this though, and I remember how easy it was to make it global, one line change!

**Comment by Xelor41023** [comment]
Maybe it would be better to duplicate it and have both....

**Comment by bobdaduck** [comment]
Alternatively: rip out all voice chat and just use discord

**Comment by raptor** [comment]
I'm seriously considering removing voice chat - has anyone used in-game voice chat recently?

**Comment by TheSkybax** [comment]
bob used it just to prove that it was there. But no, nobody actually uses it, especially now that we have a discord that almost every active person is in

**Comment by Xelor41023** [comment]
Absolutely keep it in it shouldnt be removed if bitfighter ever gets a
larger player base it will be needed.

**Comment by raptor** [comment]
It's not hurting anything at the moment, but I can only see it being used for LAN type games where internet access may be limited (or filtered).

**Comment by eykamp** [comment]
If we removed it, there should be some standard mechanism by which players can discover where the *real* voice chat is going on.  Maybe there can be a key that shows a message set by the server that has the voice chat info?

There are lots of good voice chat options available now, so how can we best support those?

**Comment by TheSkybax** [comment]
How about a key that links directly to the Discord, with a Y or N prompt? I'd just have to add something in the welcome screen about where to find the voice chats
